presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java

```diff
@@ -26,6 +26,7 @@
     private final Duration queuedTime;
     private final Optional<Duration> analysisTime;
 
+    private final int peakRunningTaskCount;
```

Comment:
nit: How about simply `peakRunningTasks` (similarly to `completedSplits`)

Reply:
Most other classes use peakRunningTaskCount, let's keep as it is

Reply:
In query stats it is `peakRunningTasks`: https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java#L58.
In `SqlQueryManager` it is `peakRunningTasks`, `peakRunningTasksStat`: https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManagerStats.java#L94

Only in `QueryStateMachine` it is `peakRunningTaskCount` (because of `currentRunningTaskCount`). 

Thus for consistency i would keep it as `peakRunningTasks`.

Reply:
`QueryStats` uses `totalTasks`first,  then `SqlQueryManagerStats` uses `peakRunningTasks` as it is constructed from `QueryStats`

Given `completedSplits` is already used here,  it makes sense to keep the same style,  but let's use `peakRunningTaskCount` in other places whenever possible.